src/servers/src/http/handler.rs

```diff
@@ -46,8 +47,22 @@ pub async fn sql(
     let start = Instant::now();
     let resp = if let Some(sql) = &params.sql {
         let query_ctx = Arc::new(QueryContext::new());
-        if let Some(db) = params.database {
-            query_ctx.set_current_schema(db.as_ref());
+        if let Some(ref db) = params.database {
```

Comment:
```suggestion
        if let Some(db) = &params.database {
```
from https://pingcap.github.io/style-guide/rust/exprs.html#match-and-if
>Prefer to let the compiler handle borrowing, rather than explicitly using ref patterns. See the [match ergonomics RFC](https://github.com/rust-lang/rfcs/blob/master/text/2005-match-ergonomics.md) for details.